data-raw/create_lookup_table.r

```diff
@@ -0,0 +1,4 @@
+
+abs_lookup_table <- scrape_abs_catalogues(refresh = FALSE)
```

Comment:
why is `refresh = FALSE` here?

Reply:
Good question. If refresh is TRUE then it updates the version loaded by the package rather than necessary return a new object. 

I am not 100% sure that this is the right approach.

Reply:
all good. I'm going to merge this PR now. We can further refine it later if needed.